New Storage: Testsuite fails when JPA Map storage is enabled for groups

### Describe the bug

When running the testsuite with `groups` configured to use the `jpa-map-storage`, the following tests fail:

- `org.keycloak.testsuite.federation.ldap.LDAPGroupMapper2WaySyncTest`
- `org.keycloak.testsuite.federation.ldap.LDAPGroupMapperSyncTest`
- `org.keycloak.testsuite.federation.ldap.LDAPGroupMapperSyncWithGroupsPathTest`

They all fail with an `IndexOutOfBoundsException`:
```
Caused by: java.lang.IndexOutOfBoundsException: Index 0 out of bounds for length 0
	at java.base/jdk.internal.util.Preconditions.outOfBounds(Preconditions.java:64)
	at java.base/jdk.internal.util.Preconditions.outOfBoundsCheckIndex(Preconditions.java:70)
	at java.base/jdk.internal.util.Preconditions.checkIndex(Preconditions.java:248)
	at java.base/java.util.Objects.checkIndex(Objects.java:372)
	at java.base/java.util.ArrayList.get(ArrayList.java:459)
	at org.keycloak.models.map.group.MapGroupAdapter.getFirstAttribute(MapGroupAdapter.java:75)
	at org.keycloak.testsuite.federation.ldap.LDAPGroupMapper2WaySyncTest.lambda$test02_syncWithGroupInheritance$26a8868a$3(LDAPGroupMapper2WaySyncTest.java:218)
```



### Version

17.0.1

### Expected behavior

Testsuite should run with no errors when `groups` are configured with the `jpa-map-storage`.

### Actual behavior

There are test failures.

### How to Reproduce?

Just fetch keycloak main, build and run the tests listed above using the JPA Map storage for `groups`.

### Anything else?

_No response_

--- a/model/map/src/main/java/org/keycloak/models/map/group/MapGroupAdapter.java
+++ b/model/map/src/main/java/org/keycloak/models/map/group/MapGroupAdapter.java
@@ -67,12 +67,7 @@ public class MapGroupAdapter extends AbstractGroupModel<MapGroupEntity> {
 
     @Override
     public String getFirstAttribute(String name) {
-        List<String> attributeValues = this.entity.getAttribute(name);
-        if (attributeValues == null) {
-            return null;
-        }
-
-        return attributeValues.get(0);
+        return getAttributeStream(name).findFirst().orElse(null);
     }
 
     @Override

